index.js

```diff
@@ -27,6 +27,18 @@ module.exports.getCredentials = function(name, plan, iname) {
         }
       }
     }
+    // Check if credentials weren't found because the service was migrated to IAM
+    for (var service_name in services) {
+      for (var i = services[service_name].length - 1; i >= 0; i--) {
+        var instance = services[service_name][i];
+        var usingResourceName = instance.credentials && instance.credentials.resource_name;
+        if ((usingResourceName && instance.credentials.resource_name === name) && 
+          (!iname || iname === instance.name)
```

Comment:
Why don't you check 
```js
if (instance.credentials.resource_name === name && (!iname || iname === instance.name)) {
  // ...
}
```

Reply:
I was just trying to be safe. In the case that `credentials` isn't a key on the object, that would cause the module to crash. But I can make that change